package.json

```diff
@@ -20,7 +19,8 @@
     "translations": "npm-run-all translations:*",
     "translations:extract": "formatjs extract 'src/**/*.js*' --out-file lang/en.json --id-interpolation-pattern '[sha512:contenthash:base64:6]'",
     "translations:compile": "formatjs compile lang/en.json --out-file compiled-lang/en.json",
-    "verify": "npm-run-all build lint test"
+    "verify": "npm-run-all build lint test",
+    "semantic-release": "semantic-release"
```

Comment:
I think there will be only 1 release command, have you considered just naming it `release`?